rln/README.md

```diff
@@ -82,14 +83,14 @@ use rln::protocol::*;
 use rln::public::*;
 use std::io::Cursor;
 
-// We set the RLN parameters: 
+// We set the RLN parameters:
 // - the tree height;
-// - the circuit resource folder (requires a trailing "/").
+// - the tree config, if it is not defined, the default value will be set
 let tree_height = 20;
-let resources = Cursor::new("../zerokit/rln/resources/tree_height_20/");
+let input = Cursor::new(json!({}).to_string());
 
 // We create a new RLN instance
-let mut rln = RLN::new(tree_height, resources);
+let mut rln = RLN::new(tree_height, input);
```

Comment:
future pr, I don't think we need to pass in the tree height as an argument anymore, we're fixing it to 20.

Reply:
Should it be added as an issue in this issue?